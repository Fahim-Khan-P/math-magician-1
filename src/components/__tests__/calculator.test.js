import React from 'react';
import { render, screen } from '@testing-library/react';
import Calculator from '../calculator';

test('Test Calculator Page is rendered', () => {
  render(<Calculator />);

  expect(screen.findByTitle('Lets do some math!')).toMatchSnapshot();
});