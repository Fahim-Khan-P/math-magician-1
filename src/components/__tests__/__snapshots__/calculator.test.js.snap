// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Calculator Page is rendered 1`] = `Promise {}`;
